hack/run-ci-e2e-test.sh

```diff
@@ -10,9 +10,34 @@ NODE_COUNT=""
 SKIP_NODE_DELETION=""
 KEY_PAIR_NAME=""
 
-
 export CGO_ENABLED=0
 
+get_wmco_logs() {
+  oc logs -l name=windows-machine-config-operator -n windows-machine-config-operator
+}
+
+# This function runs operator-sdk run --olm/cleanup depending on the given parameters
+# Parameters:
+# 1: command to run [run/cleanup]
+# 2: path to the operator-sdk binary to use
+# 3: path to the directory holding the operator manifests
+OSDK_WMCO_management() {
```

Comment:
Why is `WMCO` in capitals here but not in `get_wmco_logs()`. Let us pick one and stick with that.